tests/unit/src/test/scala/tests/codeactions/CreateNewFileLspSuite.scala

```diff
@@ -0,0 +1,79 @@
+package tests.codeactions
+
+import scala.meta.internal.metals.Messages.NewScalaFile
+import scala.meta.internal.metals.codeactions.ImportMissingSymbol
+import scala.meta.internal.metals.codeactions.CreateNewFile
+import munit.Location
+import org.eclipse.lsp4j.ShowMessageRequestParams
+import scala.meta.internal.metals.MetalsEnrichments._
+
+class CreateNewFileLspSuite extends BaseCodeActionLspSuite("createNew") {
+
+  checkNewFile(
+    "basic",
+    """|package a
+       |
+       |case class School(name: String, location: <<Location>>)
```

Comment:
Maybe let's add at least one other test case for other class kind?

Reply:
done